src/lib/exec-fetcher.test.js

```diff
@@ -1,10 +1,17 @@
 const test = require('ava')
 const path = require('path')
-const fs = require('fs-extra')
+const _fs = require('fs')
 const tempDirectory = require('temp-dir')
 const { v4: uuid } = require('uuid')
+const util = require('util')
+
 const { getExecName, shouldFetchLatestVersion, fetchLatestVersion } = require('./exec-fetcher')
 
+const fs = {
```

Comment:
Would it make sense to group all the `const fs = { ... util.promisify(...), ... }` into a single file?

```js
const fs = require('../utils/fs')
```

Reply:
Yep, this is much better.